packages/components/src/components/Spacer/Spacer.tsx

```diff
@@ -1,25 +1,45 @@
 import { View } from 'react-native'
 import React, { FC } from 'react'
 
+import { spacing } from '@department-of-veterans-affairs/mobile-tokens'
+
 export type SpacerProps = {
-  /** Size of the spacer, default 10 */
-  size?: number
+  /** Size of the spacer, default sm */
+  size?:
+    | 'none'
+    | '2xs'
+    | 'xs'
+    | 'sm'
+    | 'md'
+    | 'lg'
+    | 'xl'
+    | '2xl'
+    | '3xl'
+    | '4xl'
+    | '5xl'
+    | '6xl'
```

Comment:
@TimRoe Wanted to get your opinion on this. We want to lock the spacer size prop down to the available spacing tokens. I was thinking of either doing this by using the abbreviated spacer name as "variants" like above, OR locking it down to the explicit number values that match the spacing tokens.

### Abbreviated token  names
**Pros**
- User would not have to import spacing tokens or remember exact values
- If the spacing token values change, it wouldn't necessarily require changes to end user code or our Spacer code (they'd still have to account for adjustment in value)

**Cons**
- Token names don't mean much to the end user. They don't have much sense in what xs, s, med, etc mean
- Names don't match tokens exactly

### Number values
**Pros**
- User knows exact value, or can import spacing tokens

**Cons**
- Extra step in importing spacing tokens if doing so
- Less clear that its locked down to certain values
- If values change, will likely require code changes for end user

Curious if you had any thoughts or different approaches in mind.

Reply:
Of those two, prefer the `Abbreviated token names` route. Both cons seem minor to me. The design they're working from should be using the tokens so they don't need to care what the actual size is and lopping off `vadsSpacer` seems sensible.

I do wonder if we can use TS somehow to get the suggestion to show both. My initial thought is `enum`, but not sure if there's a way to do an enum where they wouldn't need to import it--which I think should be avoided. 

If  something with enum can't work without import, does it work to apply doc to the individual values? As in:
```
size?:
  /** 0 */
    | 'none'
  /** 4 */
    | '2xs'
```
or something like that.

Be cool if there was a way to communicate the value as well as the auto-fill abbreviated name, but if we can't find a way then as is seems optimal to me.

Reply:
Separated out SpacerSize into a separate prop and added some JSDoc links which should come in handy.

![Screenshot 2024-09-11 at 12 58 38 PM](https://github.com/user-attachments/assets/1a977383-3c13-4fc0-b5e1-65e5848b5008)